### DISCUSSION TITLE
Spotweb wipe database (to be able to retrieve old spots)

### DISCUSSION THREAD

**Discussion by rrademak** [alexbelgium/hassio-addons#1152] [General]
Hi all,

Somehow my spots only includes spots from 1 day back. Possible I've used the wrong newsserver. So while adding a new server with high retention  the database is not updates anymore with older spots. Any method to start from scratch ? seems the DB is not wiped so every time I install I'm getting a small amounts of spots.

Thx!
Ronald

**Comment by rrademak** [comment]
Found it, Installed portainer and executed upgrade-db --reset-db from the container console.

thx,
Ronald

**Comment by alexbelgium** [comment]
Thanks for taking the time to describe your solution for other users! Enjoy the add-on
